### ISSUE TITLE
Responsive Demo

### ISSUE THREAD

**Issue by radiovisual** [suitcss/suit#155]
I love working with this component, but it took me a while to finally get up and running. 

So my humble request: It would be awesome if one of the many SUIT [Sages|Wizards|Magicians|Ninjas|Unicorns] could throw together a fully responsive demo of components-grid in action (and/or point to real-world examples or online tutorials?), utilizing best practices. The readme has a passive note about responsiveness:

> Cell widths and offsets can be controlled using the [responsive sizing utilities](https://github.com/suitcss/utils-size) and [responsive offset utilities](https://github.com/suitcss/utils-offset), respectively.

Which meant a bit of trial and error and jumping around to the different plugin readmes. 

A demo could also help avoid a few common pitfalls, like if I was new to SUIT and I wanted to use a copy-paste from some of the documentation, I might end up with a simple setup like this:

```html
<div class="Grid Grid--fit Grid--withGutter">
    <div class="Grid-cell u-size12of12 u-lg-size3of12"></div>
    <div class="Grid-cell u-size12of12 u-lg-size3of12"></div>
    <div class="Grid-cell u-size12of12 u-lg-size3of12"></div>
    <div class="Grid-cell u-size12of12 u-lg-size3of12"></div>
</div>
```
...where one might have to bang their head against the wall for an hour before figuring out that the `Grid--fit` modifier might override all of the sizing utilities placed on `Grid-cell` descendants -- breaking the responsiveness of the Grid.

🦄  ❤️

**Comment by mlnmln** [comment]
@simonsmith 

I think grid is one of the more powerful and useful components. 
We had trouble with people not understanding the concepts behind it as well. 
Having better documentation and demos that help its adoption rate would be great.
I could recreate the responsive demo using the current syntax, then we'd link that in a prominent location. 

What do you think?

**Comment by oleersoy** [comment]
[Here's a responsive grid demo](https://github.com/superflycss/component-grid/blob/master/src/test/html/index.html) - Obviously feel free to copy it back to suit.  Rendered test link (It's test 2):

https://superflycss.github.io/component-grid/target/test/html/

```
{% set when1 %}
<code class="u-text-color-md-pink-400">Grid </code> is applied.
{% endset %}
{% call Test.component( 'Grid', when1, 'Cells are rendered with the sizing utility applied when the viewport is large otherwise they are full width.' ) %}
<div class="Grid Grid--withGutter">
  <div class="u-lg-size1of4">
    <div class="TestBox">1</div>
  </div>
  <div class="u-lg-size1of4">
    <div class="TestBox">2</div>
  </div>
  <div class="u-lg-size1of4">
    <div class="TestBox">3</div>
  </div>
  <div class="u-lg-size1of4">
    <div class="TestBox">4</div>
  </div>
</div>
{% endcall %}
```

**Comment by simonsmith** [comment]
> I could recreate the responsive demo using the current syntax, then we'd link that in a prominent location.

@mlnmln Yeah, I think a real world example as opposed to a test page is the way forward here

**Comment by oleersoy** [comment]
Could be this simple:
https://medium.com/@ole.ersoy/building-a-responsive-grid-with-superflycss-be6c1f28bebf
